.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,7 +13,7 @@ A clear and concise description of what the bug is.
 **Application version**
 Version number: [e.g. 1.2.3]
 
-<!-- See application version under "Mitt AtB" screen -->
+<!-- See application version under "My profile" screen -->
```

Comment:
Hva er det egentlig snakk om her, @rosvik? Sånn jeg forstår den lille setninen her vil det kanskje være mer riktig å si _"See application version at the bottom of your "My profile" page."_, i stedet for at vi snakker om "My profile screen".